packages/widget_toolkit/lib/src/lib_open_url/views/open_url_widget_with_dependencies.dart

```diff
@@ -0,0 +1,40 @@
+import 'package:flutter/material.dart';
+import 'package:provider/provider.dart';
+
+import '../../../widget_toolkit.dart';
+
+class OpenUrlWidgetWithDependencies extends StatelessWidget {
+  const OpenUrlWidgetWithDependencies({
+    required this.url,
+    required this.child,
+    this.onTap,
+    this.onError,
+    this.translateError,
+    this.uriType = UriType.https,
+    this.modalConfiguration = const ModalConfiguration(),
+    super.key,
+  });
+
+  final String url;
+  final Widget child;
+  final VoidCallback? onTap;
+  final Function(BuildContext, Exception)? onError;
+  final String Function(BuildContext, Exception)? translateError;
+  final UriType uriType;
+  final ModalConfiguration modalConfiguration;
+
+  @override
+  Widget build(BuildContext context) => MultiProvider(
```

Comment:
@nikolay-vasilev-prime here is one example whereisInTestMode should be integrated to enable the golden tests capabilities